config/network/route.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   name: controller-manager
   namespace: system
 spec:
+  host: REPLACE_HOST
```

Comment:
iinw, in OCP this value is decided by the ingress-operator (service-name-namespace-name-cluster-name)

Reply:
but that value should be in alignment with the flotta/CA certs.

Reply:
so in that case, we should pick the name when creating the cert based on expected host name, no?

Reply:
yeah, it's a parameter on the operator config, default is project-flotta.io and 127.0.0.1

Reply:
where should we pass the host here?
I tried to start yggdrasil with:
`sudo ./yggd --protocol http --path-prefix api/flotta-management/v1 --client-id $(cat /etc/machine-id) --cert-file /tmp/cert.pem --key-file /tmp/key.pem --ca-root /tmp/ca.pem --server 127.0.0.1:8043`
but received connection refused